plugins/serializers/binary/binary.go

```diff
@@ -0,0 +1,119 @@
+package binary
+
+import (
+	"encoding/binary"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"log"
+)
+
+type Serializer struct {
+	Entries    []Entry `toml:"entries"`
+	Endianness string  `toml:"endianness"`
+
+	converter binary.ByteOrder
+}
+
+func (s *Serializer) Init() error {
+	log.Printf("I [binary] binary serializer init")
+
+	if s.Endianness == "" {
+		s.Endianness = "host"
+	}
+
+	switch s.Endianness {
+	case "big":
+		s.converter = binary.BigEndian
+	case "little":
+		s.converter = binary.LittleEndian
+	case "host":
+		s.converter = internal.HostEndianness
+	default:
+		return fmt.Errorf("invalid endianness %q", s.Endianness)
+	}
+
+	for i, entry := range s.Entries {
+		if err := entry.Check(); err != nil {
+			return err
+		}
+
+		s.Entries[i] = entry
+	}
+
+	return nil
+}
+
+func (s *Serializer) Serialize(metric telegraf.Metric) ([]byte, error) {
+	serialized := make([]byte, 0)
+
+	fmt.Printf("I [binary] fields: %v\n", metric.Fields())
```

Comment:
Replace all print statements by the telegraf logger..

Also, most should be Debug level, not Inform.

Reply:
Sure, should be fixed with the latest commit